projects/plots/plots/vizapp/main.py

```diff
@@ -13,7 +13,7 @@ def main(
 
 
 def cli(args=None):
-    parser = jsonargparse.ArgumentParser(parser_mode="omegaconf")
+    parser = jsonargparse.ArgumentParser()
```

Comment:
I think we need this `parser_mode` to be able to specify environment variables in the config. Was something going wrong?

Reply:
Yeah, it was telling me that the option was found, or something like that. I meant to just remove it for testing though, I'll put it back

Reply:
You might need to install `omegaconf` if it isn't

Reply:
Oh, I didn't realize it was a separate library. Let me add that to the `pyproject.toml`.